dashboard/app/controllers/openai_evaluate_controller.rb

```diff
@@ -20,14 +25,38 @@ def evaluate
       return render status: :not_found, json: "Unit with id #{unit_id}"
     end
 
-    evaluation_type = params[:evaluationType]
-    system_prompt = AiSystemPrompts::EvaluateSystemPromptHelper.get_system_prompt(level, unit, evaluation_type)
-    student_work = prepend_system_prompt(system_prompt, params[:studentWork])
-    response = client.request_evaluation(student_work)
-    response_body = JSON.parse(response.body)
-    response_body = response_body['choices'][0]['message'] if response.code == 200
-    evaluation =  {status: response.code, json: response_body}
-    return render(status: evaluation[:status], json: evaluation[:json])
+    # If the student did not do any work on the level, we don't need to send it to AI
+    # for evaluation. Instead, return a custom response explaining why the student's
+    # work was not evaluated.
+    no_attempt_response = {
+      aiEvaluation: "No attempt",
```

Comment:
I wonder if at some point we will want to store these aiEvaluation types into a constant...

Reply:
yeah, probably at some point once they're standardized. I think eventually we'd want to make any user-facing version of them translatable too. Adding to the tidying/polish [list](https://docs.google.com/document/d/1P53DZhMELqrt0VArMsqVnN-WpCOLR7-R4mt4bBtMlGs/edit?tab=t.0)!